two/src/main.cpp

```diff
@@ -0,0 +1,176 @@
+#include <iostream>
+
+// 2
+int a = 123;
+
+void f(const int a = 5)
+{
+  std::cout << a * 2 << std::endl;
+}
+
+/*
+  this function invokes f(), which receives an integer, doubles and prints it out
+  it has a default value assigned for passing nothing to f() will print 10
+*/
+void Two()
+{
+  f(1);
+  f(a);
+  int b = 3;
+  f(b);
+  int a = 4;
+  f(a);
+  f();
+}
```

Comment:
I don't see your 'without a computer' output for the above, but I think it's fairly obvious and this looks like one of the 'low-value' exercises.